config.h

```diff
@@ -0,0 +1,13 @@
+// the configured options and settings 
+// do not edit this file
+
+
+#ifndef _CONFIG_H_
+#define _CONFIG_H_
+
+#define D2TM_PACKAGE  "d2tm"
+#define D2TM_VERSION  "0.6.X"
+#define D2TM_APP_NAME "Dune II - The Maker"
+#define D2TM_OS       "Linux OS"
```

Comment:
defines do not respect namespaces. OK, we don't use namespaces yet in this project, but modern C++ uses constexpr char* for strings like this.